tests/Application/config/routes.yaml

```diff
@@ -3,13 +3,21 @@ app_books:
         alias: app.book
         grid: app_book
         only: ['index']
+        templates: crud
```

Comment:
Does it needs `templates` config when it's a `sylius.resource_api`? Same for other routes in this file.

Reply:
yep, I forgot to transform sylius.resource_api to sylius.resource, cause we do not test the api part anymore. It's already tested with resource bundle in the package. Cause hateoas does not support attribute instead of Doctrine annotations, we'll not able to test the grid bundle with symfony 7, if we do not transform all these tests with a frontend UI.